ansible/roles/atmo-check-novnc/tasks/main.yml

```diff
@@ -89,12 +89,12 @@
     - novnc
     - check-novnc
 
-- name: fail if shell can't talk to vm
-  wait_for: >
-    search_regex=OpenSSH
-    port={{ ansible_port }}
-    host={{ ansible_host }}
-    delay=1
-    timeout={{ SSH_PORT_TIMEOUT }}
-  tags: gateone_networking_check
-  delegate_to: shell
+# - name: fail if shell can't talk to vm
+#   wait_for: >
+#     search_regex=OpenSSH
+#     port={{ ansible_port }}
+#     host={{ ansible_host }}
+#     delay=1
+#     timeout={{ SSH_PORT_TIMEOUT }}
+#   tags: gateone_networking_check
+#   delegate_to: shell
```

Comment:
We can and should delete this (in a future commit) if we don't plan to bring it back (when: some specific conditon..)